.storybook/preview.tsx

```diff
@@ -39,7 +39,9 @@ export const decorators = [
                 }
               `}
             </style>
-            <Story />
+            <div id="story-root">
```

Comment:
Added this to easily identify the div to take the snapshot of, and not the whole tree

Reply:
Could we use maybe `data-test-id`? If this is only used for testing